test/Tests/src/Semantic/Equals_Spec.enso

```diff
@@ -203,5 +205,15 @@ spec =
                 dupl_tree = tree.deep_copy
                 Test.with_clue "Seed sed to 42" (tree == dupl_tree).should_be_true
 
+    Test.group "Polyglot Operator ==" <|
+        Test.specify "should not try to compare members" <|
```

Comment:
This test actually does not test the `equalsObjectWithMembers` case, but rather `equalsHostObject` - so it invokes `equals` Java method, which, obviously, returns false.